service/worker/jobs/boptest_run_test/job.py

```diff
@@ -91,7 +99,7 @@ def run(self):
     def check_idle_time(self):
         idle_time = datetime.now() - self.last_message_time
         if idle_time.total_seconds() > self.timeout:
-            print("Testid '%s' is terminating due to inactivity." % self.testid)
+            self.logger.info("Testid '%s' is terminating due to inactivity." % self.testid)
```

Comment:
Using the logger for this stuff will ensure that we get these messages in the log file that clients are able to download.